README.md

```diff
@@ -84,11 +84,14 @@ These properties are now returned when found:
 * `docDate`: The date associated to the document.
 * `title`: The (possible) title of the document.
 * `process`: The process rules, **as they appear on the text of the document**, eg `'14 October 2005'`.
-* `deliverers`: The deliverer(s) responsible for the document (WGs, TFs, etc).
+* `deliverers`: The deliverer(s) responsible for the document (WGs, TFs, etc); an `Array` of `Object`s, each one with these properties:
+  * `homepage`: URL of the grop's home page.
+  * `name`: name of the group, exactly as it is found in the hyperlink on the document.
 * `thisVersion`: URL of this version of the document.
 * `previousVersion`: URL of the previous version of the document (the last one, if multiple are shown).
 * `latestVersion`: URL of the latest version of the document.
-* `editorIDs`: ID(s) of the editor(s) responsible for the document, necessarily matching the pattern `/\d+/`.
+* `editorIDs`: ID(s) of the editor(s) responsible for the document (an `Array` of `String`s, necessarily matching the pattern `/\d+/`).
+* `delivererID` ID of the group, (a `String`, necessarily matching the pattern `/\d+/`).
```

Comment:
Why isn't this part of the `deliverers`? Something like:
- `deliverers`
  - `id`
  - `homepage`
  - `name`

Also, there can be multiple deliverers, so there will be multiple IDs detected.